linux/snap/snapcraft.yaml

```diff
@@ -0,0 +1,62 @@
+name: nextcloud-client
+version: 2.2.4-git
+summary: nextcloud desktop client
+icon: ../client/theme/colored/owncloud-icon.svg
```

Comment:
I'm gonna go out on a limb and say you probably want something in `../nextcloudtheme/theme/colored/` :wink: .